app/models/SessionsRepository.scala

```diff
@@ -3,7 +3,7 @@ package models
 import javax.inject.Inject
```

Comment:
Make sure this file is rebased with develop properly. Some things are missing and some are added which are not required.